pyee/asyncio.py

```diff
@@ -67,7 +67,7 @@ def _emit_run(
                 return
 
             def callback(f):
-                self._waiting.remove(f)
+                self._waiting.discard(f)
```

Comment:
What does this change do?

Reply:
When we call the `cancel_all` function, the `_waiting` set has already been emptied.
the `remove` raises a `KeyError` exception, while `discard` does nothing.

Reply:
Ah, that makes sense.